rest_framework/settings.py

```diff
@@ -78,6 +78,7 @@
     # Authentication
     'UNAUTHENTICATED_USER': 'django.contrib.auth.models.AnonymousUser',
     'UNAUTHENTICATED_TOKEN': None,
+    'WWW_AUTHENTICATE_BEHAVIOR': 'first',
```

Comment:
Why isn't this a boolean, e.g. `WWW_AUTHENTICATE_ALL = False` (by default)?

Reply:
I suppose I was leaving the door open for other modes beyond `'first'` and `'all'`, but I think a boolean setting makes more sense here.